manual/source/topic/telemetry.rst

```diff
@@ -54,7 +54,7 @@ demonstration of :term:`Lisp` in an appendix to his paper
 Telemetry utilities
 -------------------
 
-The telemetry system relies on three utility programs:
+The telemetry system uses four utility programs:
```

Comment:
"uses" is not quite right here

Reply:
"There are four utility programs associated with the telemetry system:" ?